### PR TITLE
NickeManarin.ScreenToGif - Version 2.37

### PR DESCRIPTION
**Pull request by NickeManarin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59323)

### CODE REVIEW

File: manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.37.0
```
Comment (Trenly) [inline manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.installer.yaml:5]:
```suggestion
PackageVersion: 2.37
```
This should match control panel

File: manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.37.0
```
Comment (Trenly) [inline manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.37
```
This should match control panel

File: manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.locale.pt-BR.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.37.0
```
Comment (Trenly) [inline manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.locale.pt-BR.yaml:5]:
```suggestion
PackageVersion: 2.37
```
This should match control panel

File: manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.37.0
```
Comment (Trenly) [inline manifests/n/NickeManarin/ScreenToGif/2.37.0/NickeManarin.ScreenToGif.yaml:5]:
```suggestion
PackageVersion: 2.37
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
Update version folder too.

**Comment by jedieaston** [comment]
Hot tip to change the folder name: Press "." on your keyboard when you have GitHub open to this PR in your browser, and a web version of VSCode will launch.
 
 Once it loads, find the files you added, and rename the folder. Then use the source control button on the left to commit (it's automatically pushed).
 
 image

**Comment by NickeManarin** [comment]
@jedieaston Oh, I did but I forgot to push the changes after the commit. :(

**Review by ghost — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**